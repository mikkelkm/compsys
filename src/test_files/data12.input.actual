test_files/data12.input: ASCII text
