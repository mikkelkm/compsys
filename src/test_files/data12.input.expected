test_files/data12.input: data
